src/components/BaseMiniContextMenuItem.tsx

```diff
@@ -36,16 +37,18 @@ type BaseMiniContextMenuItemProps = {
      * Can be used to control the click event, and for example whether or not to lose focus from the composer when pressing the item
      */
     shouldPreventDefaultFocusOnPress?: boolean;
+
+    /**
+     * Reference to the outer element
+     */
+    ref?: ForwardedRef<View>;
```

Comment:
Maybe instead of using `ForwardedRef<View>` and later casting it would be better to use `PressableRef` 🤔 @blazejkustra, as TypeScript expert, what do you think?